internal/services/monitor/monitor_action_group_resource_test.go

```diff
@@ -300,6 +300,38 @@ func TestAccMonitorActionGroup_disabledUpdate(t *testing.T) {
 	})
 }
 
+func TestAccMonitorActionGroup_locationBasic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_monitor_action_group", "test")
+	r := MonitorActionGroupResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.locationBasic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("location").HasValue("global"),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("location").HasValue("global"),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.locationBasic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("location").HasValue("global"),
+			),
+		},
+		data.ImportStep(),
+	})
+}
```

Comment:
Since this property is `ForceNew` applying multiple configurations in the test will just end up recreating the resource at every single apply which isn't testing the new behaviour that's being added, that location can be set to a value other than `global`.

We should test that location can be set to something other than `global` since that's the default.

```suggestion
func TestAccMonitorActionGroup_location(t *testing.T) {
	data := acceptance.BuildTestData(t, "azurerm_monitor_action_group", "test")
	r := MonitorActionGroupResource{}

	data.ResourceTest(t, r, []acceptance.TestStep{
		{
			Config: r.location(data),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
			),
		},
		data.ImportStep(),
	})
}
```